configure.ac

```diff
@@ -1102,9 +1102,11 @@ if test "X$HDF5_DOXYGEN" = "Xyes"; then
   AC_SUBST([DOXYGEN_HTML_FOOTER])
   AC_SUBST([DOXYGEN_HTML_EXTRA_STYLESHEET])
   AC_SUBST([DOXYGEN_HTML_EXTRA_FILES])
+  AC_SUBST([DOXYGEN_TAG_FILE])
```

Comment:
These same changes need to be made to the CMake locations.

Reply:
Done.

Reply:
The CMake files are in c++/src/CMakeLists.txt and src/CMakeLists.txt. I can do that afterwards if you want.

Reply:
Don't forget DOXYGEN_PREDEFINED

Reply:
I think the exact same changes can be made to c++/src/CMakeLists.txt.

Reply:
And DOXYGEN_EXAMPLES_DIRECTORY.